automatic/brave/README-release.md

```diff
@@ -10,4 +10,6 @@ Brave is a free and open-source web browser developed by Brave Software Inc. bas
 
 ## Notes
 
-* This is an **official release version** of Brave. It is in continuous development with new releases landing approximately every three weeks. 
+- This is an **official release version** of Brave. It is in continuous development with new releases landing approximately every three weeks.
+
```

Comment:
Please remove this empty line, as it won't show up as expected once pushed to CCR.